test/test-ping.coffee

```diff
@@ -3,37 +3,65 @@ http        = require 'http'
 querystring = require 'querystring'
 {spawn}     = require 'child_process'
 
+flush_assertions = []
+finish_flush = null
+
 server = http.createServer (req, res) ->
   params = parse req.url, true
   if params.path is '/endpoint'
+    console.log "endpoint hit"
     req.on 'data', (chunk) ->
       data = querystring.parse chunk.toString()
       hits = JSON.parse data.json
-      if hits.one is 1 and hits.two is 2
-        console.log 'Test Succeeded'
+      if flush_assertions.shift() hits
```

Comment:
The way `flush_and_assert` is written, it looks like pushing an assertion is optional, but the way this is written, it looks like we're expecting an assertion.

Reply:
Fixed!